src/resolvers/__tests__/connection-test.js

```diff
@@ -140,26 +140,38 @@ describe('connection() resolver', () => {
       await user2.save();
     });
 
-    it('should return Resolver object', () => {
+    it('should return Resolver object with default name', () => {
       const resolver = connection(UserModel, UserTC);
+      if (!resolver) throw new Error('Connection resolver is undefined');
       expect(resolver).toBeInstanceOf(Resolver);
+      expect(resolver.getNestedName()).toEqual('connection');
+    });
+
+    it('should return Resolver object with custom name', () => {
+      const resolver = connection(UserModel, UserTC, {
+        // $FlowFixMe
```

Comment:
I did not found a solution to fix this and the error thrown by flow also does not make sense to me. Any idea or fix appreciated,